.env.dev

```diff
@@ -1,2 +1,2 @@
-export REACT_APP_BASE_URL=/
-export REACT_APP_TARGET_URL=http://0.0.0.0:3000
+REACT_APP_BASE_URL=/
+REACT_APP_TARGET_URL=http://0.0.0.0:3000
```

Comment:
why does this file change? (why remove the `export`, isn't this needed for `source .env.dev`?

Reply:
Thats not how this is to be used.

This is a template environment file.

Reply:
If you use `source` you're setting that environment variable to your entire OS. Where as using `dotenv` lets you just have a file called `.env` where it pulls those variables into context

Reply:
using `source` sets it for your shell, not the entire OS. also the README explains using source. if you want to change this then also change the instructions in the readme please.

Reply:
Its been set up from the beginning to use the `dotenv` package as most JS frameworks use this workflow in development. 

I'll update the readme accordingly

Reply:
makes sense to me, just important to have the readme contain the correct steps to get up and running!